### PR TITLE
FEAT: use 7-character slug for question

### LINKED ISSUE #46: Shorter nanoid for question
**Is your feature request related to a problem? Please describe.**
When submitting question, we generate a random id using nanoid, but it still generating too long ID (around 21char).

Even though it is better, I guess for our case it's better to generate shorter id instead.

**Describe the solution you'd like**

See https://github.com/steven-tey/dub/blob/main/packages/utils/src/index.ts#L65-L68

### PR DESCRIPTION
**Pull request by theodorusclarence**
Closes #46 

image

https://github.com/mazipan/tanyaaja/blob/4e77ac6726ff186572978abc9fb1c69baa713079/src/lib/firebase.ts#L174-L194

this function isn't used anymore right? I didn't change them

### CODE REVIEW

**Comment by mazipan** [comment]
YES, previously I am using Fire Store, but switched to Notion as database.
So it's safe to just ignore it.

File: src/lib/utils.ts [hunk 1]
```diff
@@ -149,3 +150,12 @@ export const randomizeAvatar = () => {
   const randomIndex = Math.floor(Math.random() * RANDOM_AVATARS.length)
   return RANDOM_AVATARS[randomIndex]
 }
+
+export function generateNanoId(size?: number) {
```
Comment (mazipan) [inline src/lib/utils.ts]: how about just assign the default?

```suggestion
export function generateNanoId(size = 7) {
```

Comment (theodorusclarence) [reply src/lib/utils.ts]: sure, that would be great, omw

**Review by mazipan — APPROVED** [review]
Looks solid!

**Pull request merged**